source/common/upstream/load_balancer_impl.h

```diff
@@ -454,19 +455,24 @@ class LeastRequestLoadBalancer : public EdfLoadBalancerBase {
             least_request_config.has_value()
                 ? PROTOBUF_GET_WRAPPED_OR_DEFAULT(least_request_config.value(), choice_count, 2)
                 : 2),
-        active_requests_exponent_runtime_(
-            least_request_config.has_value() && least_request_config->has_active_requests_exponent()
+        active_request_bias_runtime_(
+            least_request_config.has_value() && least_request_config->has_active_request_bias()
                 ? absl::optional<Runtime::Double>(
-                      Runtime::Double(least_request_config->active_requests_exponent(), runtime))
+                      Runtime::Double(least_request_config->active_request_bias(), runtime))
                 : absl::nullopt) {
     initialize();
   }
 
 protected:
   void refresh(uint32_t priority) override {
-    active_requests_exponent_ = active_requests_exponent_runtime_.has_value()
-                                    ? active_requests_exponent_runtime_->value()
-                                    : 1.0;
+    active_request_bias_ =
+        active_request_bias_runtime_.has_value() ? active_request_bias_runtime_->value() : 1.0;
+
+    if (active_request_bias_ < 0.0) {
+      ENVOY_LOG(warn, "upstream: invalid active request bias supplied (runtime key {}), using 1.0",
```

Comment:
Might want to also validate it shows the log message in your unit test. Something like:
https://github.com/envoyproxy/envoy/blob/841ad99de4945742b5e8a81e808d252f6a5cb7af/test/extensions/common/utility_test.cc#L65

If for some reason TSAN builds start failing after you add that test, protect this thing with a mutex and that should fix it:
https://github.com/envoyproxy/envoy/blob/841ad99de4945742b5e8a81e808d252f6a5cb7af/test/test_common/logging.h#L61

Reply:
Arguably this could also be a stat. Generally I think we prefer having stats to accompany warnings as logs aren't necessarily tied to operations alerting. OTOH, this particular case is probably going to be pretty rare/unlikely if ever, so I don't think we need to do a stat.